meta/main.yml

```diff
@@ -8,8 +8,14 @@ galaxy_info:
   platforms:
     - name: EL
       versions:
-        - all
+        - 6
+        - 7
+    - name: Ubuntu
+      versions:
+       - xenial
+       - trusty
   categories:
     - system
+
 dependencies:
-  - config_encoder_filters
+  - jtyr.config_encoder_filters
```

Comment:
Please remove this change. To install the `config_encoder_filters` role, please follow the [installation instructions](https://github.com/jtyr/ansible-config_encoder_filters#installation).